components/changelog/ChangelogTrigger.js

```diff
@@ -62,10 +62,22 @@ const ChangelogTrigger = props => {
   return (
     <Flex>
       {hasSeenNewUpdates ? (
-        <FlameIcon onClick={handleShowNewUpdates} backgroundColor="black.100" url="/static/images/flame-default.svg" />
+        <FlameIcon
+          height={height}
+          width={width}
+          onClick={handleShowNewUpdates}
+          backgroundColor="black.100"
+          url="/static/images/flame-default.svg"
+        />
       ) : (
         <Dropdown>
-          <FlameIcon onClick={handleShowNewUpdates} backgroundColor="yellow.100" url="/static/images/flame-red.svg" />
+          <FlameIcon
+            height={height}
+            width={width}
+            onClick={handleShowNewUpdates}
+            backgroundColor="yellow.100"
+            url="/static/images/flame-red.svg"
+          />
```

Comment:
@ersachin3112 : I noticed that the design for the background frame icon for smaller screens is smaller than the regular background icon; the width and height should be, `9.49px` and `13.5px` respectively. I think you can easily add this by adding a new prop maybe for `backgroundSize`. So it will look like,

![image](https://user-images.githubusercontent.com/12435965/124620349-3bea3a00-de2e-11eb-90a2-46f702c2a32a.png)

instead of the kinda oversized flame icon which we have now,

![image](https://user-images.githubusercontent.com/12435965/124620476-53c1be00-de2e-11eb-9c79-ff4809210356.png)

Reply:
I've pushed this change. 😄